core/sql/udrserv/UdrCfgParser.cpp

```diff
@@ -44,22 +44,15 @@ NABoolean UdrCfgParser::cfgFileIsOpen(NAString &errorText)
 
    NABoolean envFound = FALSE;
 
-   if(cfgFileName = getenv("MXUDRCFG"))
+   if(cfgFileName = getenv("TRAFUDRCFG"))
    {
       envFound = TRUE;
-      UDR_DEBUG1("UdrCfgParser(): MXUDRCFG cfgFileName is %s", cfgFileName);
+      UDR_DEBUG1("UdrCfgParser(): TRAFUDRCFG cfgFileName is %s", cfgFileName);
    }
    else
    {
-     NAString s("c:/tdm_sql/udr/mxudrcfg");
-     char installdir[1024];
-     Lng32 resultlength = 0;
-     Int32 res = ComRtGetInstallDir(installdir, 1024, &resultlength);
-     if (res == 0)
-     {
-       s = installdir;
-       s += "/mxudrcfg";
-     }
+     NAString s(getenv("MY_SQROOT"));
```

Comment:
What does NAString ctor do if passed a null pointer? (That is, what happens if $MY_SQROOT is not defined? Unlikely, I know.)